packages/app-store/routing-forms/pages/routing-link/[...appPages].tsx

```diff
@@ -213,8 +209,6 @@ export default function RoutingLink(props: inferSSRProps<typeof getServerSidePro
   return <RoutingForm {...props} />;
 }
 
-RoutingLink.isBookingPage = true;
```

Comment:
not needed anymore as we handle this using `useIsBookingPage` hook in app-providers-app-dir.tsx